README.md

```diff
@@ -15,26 +15,75 @@ Once you have implemented the `sa` program (and tested it to the best of your ab
+when translated to C. 
+Some consepts I have a better understanding of include pointers, 
+structs, iterator pattern, header files, char as int, and strings. <br>
+Some insights regarding Windows vs. Linux became clear, when we experienced problem, as described below. 
 
-*Which algorithm did you use to construct the suffix array. What is its running time?*
+## Problems encountered if any
+Due to privious projects not being finished, this project had to be made alongside those. 
+We chose to focus on radix sort rather than more complicated algorithms like skew and SAIS to have more time.
+Radix sort and binary search were not problematic in themselves, which gave us time to focus on the implementation.
 
+The group added a new member, Gaia, who had almost no prior experience with writing C. 
+Because of this, the first meeting of Gaia and Jonathan was focused on getting Gaia ready 
+by running code from project 1. Running this on a Windows mashine (priviously only run on Linux and Mac) 
+uncovered two problems in the code. 
+1. Since Windows saves line breaks as \r\n a line break in files read with "r" will be counted as two chars,
+which will make the string length longer than expected. This was solved by reading the file in binary mode "rb". 
+2. When making the strict border array, the code was missing a check for whether bax[i]==0. Without this, the code tried 
+to find index -1 of bax. By coincidence index -1 pointed to a place in memory which contained 0, the desired value, 
+in every test. Probably since there was some relation between the location of bax and some other value. 
+In Linux that is. When run on Windows, index -1 gave a very small negative number. This is not a problem with Windows, 
+but an implementation error only discovered, because we ran it on Windows. 
 
+The problems show the importance of running code on different OS, both to test whether it can run and to uncover errors
+which were hidden by sheer coincidence. 
 
+The week after this meeting each group member focused on a different project. Anna focused on project 1, 
+Jonathan on project 2, and Gaia on project 3. Lastly, we used one full day together with project 3's code. 
 
-## Insights you may have had while implementing the algorithm
+We were close to not finish the code before deadline. Unfortunately we did not finish the report within this time. 
+One problem we had was timing of the algorithms. We had problems with the editor being struck, since we made a lot of files.
+A lot of time was wasted, waiting for the editor to index the files, only to crash. When we changed our strategy another 
+problem became apperent. Since we did not have the time or knowledge to make proper use of free() we would quickly run 
+out of memory when trying to run our tests, since we wanted to collect data from different n and m. We ran smaller 
+batches to compensate. Ideally, we would have fixed it with better memory control. 
```

Comment:
Memory management is important in C. You can sometimes get away with not freeing memory, but not for large data. You ideally want to think it in from the beginning, because fixing it later is damned near impossible.